mod-source-record-manager-server/src/main/java/org/folio/services/JobExecutionService.java

```diff
@@ -114,4 +114,7 @@ public interface JobExecutionService {
    */
   Future<Boolean> completeJobExecutionWithError(String jobExecutionId, OkapiConnectionParams params);
 
+
+  Future<Boolean> deleteJobExecutionByIds(List<String> ids, String tenantId);
```

Comment:
The same with method renaming

Reply:
@SerhiiNosko - PR updated as per comments, Thanks 🙂